src/components/common/FileUploadPdf.vue

```diff
@@ -162,6 +162,7 @@ export default class FileUploadPdf extends Vue {
     // try to retrieve file info
     const fileInfo = await this.retrieveFileInfo(file).catch(() => null as PdfInfoIF)
     if (fileInfo === null) {
+      console.log('Error: failed to retrieve file info') // eslint-disable-line no-console
```

Comment:
Great great addition Sev. This is exactly where I found where to go and what was causing the issue.